server/controllers/profile.js

```diff
@@ -38,4 +38,20 @@ exports.loadProfile = asyncHandler(async (req, res, next) => {
       profile: profile,
     },
   });
+});
+
+// @route GET /profiles  
+// @desc Get user profiles
+// @access Public
+
+exports.loadProfiles = asyncHandler(async (req, res, next) => {
+  const loadProfiles = await Profile.find({})
+ 
+  if (!loadProfiles) {
```

Comment:
`loadProfiles` returns a list, this 'if' clause will evaluate to false even if no profiles are found as an empty list is a truthy value.

Instead lets return the empty list to the FE to have a consistent response format (we can conditionally display a message on the FE in the absence of profiles)